### PR TITLE
Add initial 3 CDI alert runbooks

### PR DESCRIPTION
**Pull request by akalenyu**
- CDIOperatorDown
- CDINotReady
- CDIDataVolumeUnusualRestartCount

Signed-off-by: Alex Kalenyuk <akalenyu@redhat.com>

### CODE REVIEW

**Comment by akalenyu** [comment]
/hold
figuring out how to edit cmds to play nicely with null values

File: docs/runbooks/CDIOperatorDown.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# CDIOperatorDown
+
+## Meaning
+
+The CDI Operator is responsible for deploying and managing the CDI infrastructure components, such as the datavolume/PVC controllers.
```
Comment (sradco) [inline docs/runbooks/CDIOperatorDown.md:5]: @akalenyu Please add this description of what CDI is to the other alert runbooks.

File: docs/runbooks/CDINotReady.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Meaning
+
+The CDINotReady alert is indicative of a CDI installation being in a degraded state;  
+- Not progressing
+- Not available to use
+
+## Impact
+
+CDI is simply not usable, its components are not ready & stopped progressing towards a ready state and hence require action.
```
Comment (sradco) [inline docs/runbooks/CDINotReady.md:11]: @akalenyu Please update what it means its not usable, what can't the user/system do if this happens?

File: docs/runbooks/CDIOperatorDown.md (hunk 1 above)
Comment (akalenyu) [reply docs/runbooks/CDIOperatorDown.md]: Done

File: docs/runbooks/CDINotReady.md (hunk 2 above)
Comment (akalenyu) [reply docs/runbooks/CDINotReady.md]: Done

**Pull request merged**